### PR TITLE
Add Cmake build process and sassy readme

### PR DESCRIPTION
**Pull request by kutlay**
I can finally play it!

I cleaned up the repository a little so it's a little likely that things may break for you. 

Likely important things:

- Tried to put all assets into one place. Now we have an assets folder with the "retina" stuff from before. This made the ship twice as big.
- Changed the SDL_image import to be SDL_image.h directly. Internet says this should work for you as well. Let me know if it breaks.
- Added a sassy readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+## How to build like a normal human being
+
+Open Xcode and click the run button.
+
+## How to build if you hate yourself
+
+### Requirements
+
+You need `conan`. If you're using a Mac:
```
Comment (Andy-Guti) [inline README.md]:
```suggestion
You need `conan` and `cmake`. If you're using a Mac:
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+```
+brew install conan
+conan profile detect
+```
```
Comment (Andy-Guti) [inline README.md]:
```suggestion
brew install conan
conan profile detect

brew install cmake
```
```

**Review by Andy-Guti — COMMENTED** [review]
Added cmake as a requirement in the README because it wasn't working for me until I installed it. Other than that everything looks good!

**Comment by kutlay** [comment]
Amazing, glad it worked

**Pull request merged**